src/api/app/models/kiwi/image.rb

```diff
@@ -157,6 +158,15 @@ def self.binaries_available(project, use_project_repositories, repositories)
 
   private
 
+  def kiwi_indentation(xml)
+    content = xml.xpath('image').children.first.try(:content)
+    if content
```

Comment:
In this case I would use the ternary operator ```?```, because you are returning a value, no running a method like in my other comment about using an ```if else end```